rules/windows/defense_evasion_masquerading_renamed_autoit.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/09/01"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious AutoIt process execution was detected. Malware written as AutoIt scripts tend to rename the AutoIt main
+executable to avoid attention. Verify process details such as command line, network connections, file writes and parent
+process details as well.
```

```suggestion
Identifies a suspicious AutoIt process execution. Malware written as AutoIt scripts tend to rename the AutoIt
executable to avoid detection. 
```
Could move to `note` - 
> Verify process details such as command line, network connections, file writes, and parent
process details.